src/danog/MadelineProto/MTProto.php

```diff
@@ -478,8 +487,23 @@ class MTProto extends AsyncConstruct implements TLCallback
         'full_chats' => 'array',
         'channel_participants' => 'array',
         'usernames' => 'array',
+        'session' => 'arrayNullCache'
```

Comment:
How about somthing like this?

```suggestion
        'session' => [
            'type' => 'array', 
            'config' => ['enable_cache'=>false]
         ]
```

Reply:
Good idea!